editoast/src/models/paced_train.rs

```diff
@@ -71,6 +71,20 @@ impl PacedTrain {
             options: self.options,
         }
     }
+
+    pub fn iter_occurrences(self) -> Vec<TrainSchedule> {
```

Comment:
could we have

```suggestion
    pub fn iter_occurrences(&self) -> impl Iterator<Item = TrainSchedule> {
```

Reply:
moved to https://github.com/OpenRailAssociation/osrd/pull/11355#discussion_r2053743491